packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -601,23 +601,27 @@ class _SliverAppBarDelegate extends SliverPersistentHeaderDelegate {
       maxExtent: maxExtent,
       currentExtent: math.max(minExtent, maxExtent - shrinkOffset),
       toolbarOpacity: toolbarOpacity,
-      child: new AppBar(
-        leading: leading,
-        automaticallyImplyLeading: automaticallyImplyLeading,
-        title: title,
-        actions: actions,
-        flexibleSpace: flexibleSpace,
-        bottom: bottom,
-        elevation: forceElevated || overlapsContent || (pinned && shrinkOffset > maxExtent - minExtent) ? elevation ?? 4.0 : 0.0,
-        backgroundColor: backgroundColor,
-        brightness: brightness,
-        iconTheme: iconTheme,
-        textTheme: textTheme,
-        primary: primary,
-        centerTitle: centerTitle,
-        titleSpacing: titleSpacing,
-        toolbarOpacity: toolbarOpacity,
-        bottomOpacity: pinned ? 1.0 : (visibleMainHeight / _bottomHeight).clamp(0.0, 1.0),
+      child: new MediaQuery.removePadding(
```

Comment:
Discussed offline, move this to line 961

Reply:
Done.